### PR TITLE
fix(peek): print error if lsp is unable to get file contents

### LINKED ISSUE #2369: Peek definition fails in C#
### Problem description

1. Open a C# file.
2. Move the cursor over a method call.
3. Press Space, l, p, d. (LSP Peek Definition).

An error message appears:

> Error executing vim.schedule lua callback: /home/gebb/.local/share/lunarvim/lvim/lua/lvim/lsp/peek.lua:42: 'width' key must be a positive Integer
stack traceback:
        [C]: in function 'nvim_open_win'
        /home/gebb/.local/share/lunarvim/lvim/lua/lvim/lsp/peek.lua:42: in function 'create_floating_file'
        /home/gebb/.local/share/lunarvim/lvim/lua/lvim/lsp/peek.lua:71: in function 'handler'
        /usr/share/nvim/runtime/lua/vim/lsp.lua:964: in function 'cb'
        vim.lua:285: in function <vim.lua:285>

### LunarVim version

5ee460f  chore(release): update changelog

### Neovim version (>= 0.6.1)

NVIM v0.6.1 Build type: Release LuaJIT 2.1.0-beta3

### Operating system/version

Arch Linux

### Relevant log output

```shell
:checkhealth
Nothing suspicious.
```

### Screenshots

LSP info:
image

### PR DESCRIPTION
**Pull request by abzcoding**
# Description

Print an error in case LSP is unable to get contents

Fixes #2369

### CODE REVIEW

File: lua/lvim/lsp/peek.lua [hunk 1]
```diff
@@ -32,6 +32,10 @@ local function create_floating_file(location, opts)
     math.min(range["end"].line + 1 + (opts.context or 10), range.start.line + (opts.max_height or 15)), -- Don't let the window be more that 15 lines long(height)
     false
   )
+  if next(contents) == nil then
+    print "peek: Unable to get contents of the file!"
```
Comment (MaxVerevkin) [inline lua/lvim/lsp/peek.lua:36]: Perhaps use `vim.notify`?

Comment (abzcoding) [reply lua/lvim/lsp/peek.lua:36]: there are a couple of more print statements in that file, so we might need to create a general output function that uses vim.notify if available and print as a backup, but that will be in another PR 😅 

cc: @kylo252

Comment (abzcoding) [reply lua/lvim/lsp/peek.lua:36]: no, users can disable it if they want
```lua
lvim.builtin.notify.active = false
```

Comment (MaxVerevkin) [reply lua/lvim/lsp/peek.lua:36]: But vim.nortify is a built-in function, notify plugin just overrides it?

Comment (abzcoding) [reply lua/lvim/lsp/peek.lua:36]: yes you are right on that front, ok let me replace all of those print statements with vim.notify

**Pull request merged**